src/ArticlePage.jsx

```diff
@@ -1,15 +1,16 @@
 import { useLocation } from "react-router-dom";
-import { getArticleById } from "../API";
+import { getArticleById, getCommentsByArticleId } from "../API";
 import { useEffect, useState } from "react";
 import "./cssFiles/ArticlePage.css";
 
 export default function ArticlePage() {
   const location = useLocation();
   const { id } = location.state;
   const [individualArticle, setIndividualArticle] = useState([]);
+  const [comments, setComments] = useState([]);
 
   function formatDate(isoString) {
-    const options = { year: 'numeric', month: 'long', day: 'numeric' };
+    const options = { year: "numeric", month: "long", day: "numeric" };
     return new Date(isoString).toLocaleDateString(undefined, options);
```

Comment:
This seems cool! I would change `undefined` to be `null` as a good practice, null shows it is intentionally left out